src/raftstore/store/fsm/store.rs

```diff
@@ -0,0 +1,1256 @@
+// Copyright 2016 PingCAP, Inc.
```

Comment:
2018

Reply:
Actually the file is moved and `peer.rs` is a new file.